schemas/applications/schema/joint_positions.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Joint Positions",
+  "description": "A description of the joint configurations available in the application",
+  "type": "object",
+  "additionalProperties": false,
+  "patternProperties": {
+    "^[a-z]([a-z0-9_]?[a-z0-9])*$": {
+      "title": "Joint Positions",
+      "description": "The names and positions of a static joint configuration in the application",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "names": {
+          "title": "Joint Names",
+          "description": "The names of the joints in the same order as the positions",
+          "type": "array",
+          "minItems": 1,
+          "uniqueItems": true,
+          "items": "string"
+        },
+        "positions": {
+          "title": "Joint Positions",
+          "description": "The positions of the joints in the same order as the names",
```

Comment:
I was wondering if we should add the units here ([m] for prismatic, [rad] for revolute)